notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -1416,7 +1433,9 @@
     "    band = bands[band_ind]\n",
     "    band_slice = band_slices[band_ind]\n",
     "    if freq_decimation > 1:\n",
-    "        band_slice = slice(band_slice.start, band_slice.stop, freq_decimation)\n",
+    "        band_slice = slice(band_slice.start + CORR_MATRIX_FREQ_OFFSET,\n",
+    "                           band_slice.stop - CORR_MATRIX_FREQ_OFFSET, \n",
```

Comment:
Should this not be `band_slice.stop + CORR_MATRIX_FREQ_OFFSET`?

Reply:
This was a defensive move to keep the slice away from the edges, in case some fully flagged channels leaked in somehow. If there is no chance of fully flagged channels (or if we want fully flagged channels to error), we can just do away with this new behavior.

Reply:
Within a band, the Nsamples/Flags should be uniform over channels. If not, something went wrong in the LST-binner (the whole idea of inpainting is to enable this). So, if that is the only motivation for this argument, I'd say we can remove it.